public/js/actions/breakpoints.js

```diff
@@ -1,21 +1,29 @@
+// @flow
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-"use strict";
 
 const constants = require("../constants");
 const { PROMISE } = require("../util/redux/middleware/promise");
 const { getBreakpoint, getBreakpoints } = require("../selectors");
 const { Task } = require("../util/task");
 const fromJS = require("../util/fromJS");
 
-function enableBreakpoint(location) {
+import type { Location } from "./types";
```

Comment:
Nope, the syntax for types is only ES6 :/ But that's because CommonJS doesn't really have a way to do this. You'd have to special syntax for CommonJS, which starts getting weird because usually `require` is just an expression. So I get their design choice.

ES6 is the future anyway. Since we're using a build step, I wouldn't be surprised if we converted to ES6 modules at some point. But I don't think mixing the styles is too bad.